### PR TITLE
Issue 51 order manager

### PR DESCRIPTION
**Pull request by bodopeti**
New endpoint added which returns approve url to given order (identified by order id). This url is acquired from Paypal trough Payment manager. Payment request is filled with static data, because reservations are not stored in database yet.

### CODE REVIEW

File: adele-order-manager/src/main/java/com/microservicesteam/adele/ordermanager/domain/OrderService.java [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
 
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 
+@Slf4j
 @Service
 @RequiredArgsConstructor
 public class OrderService {
 
+    private static final String URL = "/orders/%s/payment?success=%s";
```
[the remark this answers is not in the document]
Comment (bodopeti) [reply adele-order-manager/src/main/java/com/microservicesteam/adele/ordermanager/domain/OrderService.java]: This URL will be called by Paypal so it has to contain the whole path. This path will be environment specific. Let's discuss: how should be this URL built.

File: pom.xml [hunk 2]
```diff
@@ -79,6 +79,11 @@
                 <artifactId>adele-init</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```
Comment (bearcheese) [inline pom.xml]: Could you please also add `adele-order-manager` as a managed dependency here, and furthermore add it to the `adele-application` pom as a dependency please 😜

File: adele-order-manager/src/main/java/com/microservicesteam/adele/ordermanager/domain/OrderService.java [hunk 3]
```diff
@@ -20,6 +35,22 @@ public String saveOrder(PostOrderRequest orderRequest) {
...
     }
 
+    public ApproveUrlResponse initiatePayment(String orderId) {
+        PaymentResponse paymentResponse = paymentManager.initiatePayment(createPaymentRequest(orderId));
+
+        if (paymentResponse.status().equals(PaymentStatus.FAILED)) {
+            throw new InvalidPaymentResponseException("Payment initiation failed to orderId: " + orderId);
+        }
+
+        String paymentID = paymentResponse.paymentId().orElseThrow(() -> new InvalidPaymentResponseException("Payment id missing to orderId: " + orderId));
```
Comment (bearcheese) [inline adele-order-manager/src/main/java/com/microservicesteam/adele/ordermanager/domain/OrderService.java]: Minor: `paymentId`

File: adele-order-manager/src/main/java/com/microservicesteam/adele/ordermanager/domain/OrderService.java [hunk 4]
```diff
@@ -1,17 +1,32 @@
...
+import com.microservicesteam.adele.payment.Ticket;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 
+@Slf4j
 @Service
 @RequiredArgsConstructor
 public class OrderService {
 
+    private static final String URL = "/orders/%s/payment?status=%s";
```
Comment (kelemena) [inline adele-order-manager/src/main/java/com/microservicesteam/adele/ordermanager/domain/OrderService.java]: The URL should contain http://domain as well as this is called by Paypal. 
We should import this parameter from config.

File: adele-order-manager/src/test/java/com/microservicesteam/adele/ordermanager/domain/OrderServiceTest.java [hunk 5]
```diff
@@ -1,18 +1,26 @@
 package com.microservicesteam.adele.ordermanager.domain;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.Matchers.any;
+
+import static org.mockito.Mockito.*;
```
Comment (bearcheese) [inline adele-order-manager/src/test/java/com/microservicesteam/adele/ordermanager/domain/OrderServiceTest.java]: Can we avoid start import pls?

**Pull request merged**